scipy/signal/filter_design.py

```diff
@@ -2027,6 +2043,7 @@ def bilinear(b, a, fs=1.0):
     >>> plt.ylabel('Magnitude [dB]')
     >>> plt.grid()
     """
+    _validate_fs(fs)
     fs = float(fs)
```

Comment:
So at least some places it cannot be None, like here. I'd rather make this:
```suggestion
    fs = _validate_fs(fs)
```
and add a `allow_none=False` or `True` (depending on how common allowing `fs=None` is)

Reply:
(and inside `_validate_fs`, if it's non-None, cast to float -- and in either case, return it)

Reply:
Hey @larsoner , thanks for the comments. I incorporated the explicit check for None and also added the cast to float and a return statement.